vite.config.js

```diff
@@ -1,7 +1,20 @@
 import { defineConfig } from "vite";
 import { svelte } from "@sveltejs/vite-plugin-svelte";
+import path from "path";
 
 // https://vitejs.dev/config/
+/** @type {import('vite').UserConfig} */
 export default defineConfig({
     plugins: [svelte({ useVitePreprocess: true })],
+    resolve: {
+        alias: {
+            $src: path.resolve("./src"),
+            $base_components: path.resolve("./src/base_components"),
+            $assets: path.resolve("./src/assets"),
+            $lib: path.resolve("./src/lib"),
+            $navigation: path.resolve("./src/lib/navigation"),
```

Comment:
Same thing as above. I don't think we want to be updating our vite config every time we add a new path...